### PR TITLE
Update sync limitations: multiple sync attempts

### PR DESCRIPTION
**Pull request by govorov**

### CODE REVIEW

File: content/refguide/offline-first.md [hunk 1]
```diff
@@ -117,11 +117,21 @@ If you have custom widgets or JavaScript actions which use an entity that cannot
 
 {{% image_container width="450" %}}![custom synchronization](attachments/offline-first/custom-sync.png){{% /image_container %}}
 
-### 2.5 Error Handling {#error-handling}
+### 2.5 Limitations
+
+Running multiple synchronization processes at the same time is not supported, regardless the type (full/selective).
+
+If you try to trigger another synchronization process while the synchronization is in progress, the error message will be shown:
```
Comment (omnajjar) [inline content/refguide/offline-first.md:124]: I think here we should only outline the limitation and provide a link to the synchronize action limitation section for extra details about it., something like: 

> Running multiple synchronization processes at the same time is not supported, for more details [link_to_action]

File: content/refguide/synchronize.md [hunk 2]
```diff
@@ -68,7 +68,17 @@ The **Action** section of the properties pane shows the action associated with t
 
 {{% snippet file="refguide/microflow-common-section-link.md" %}}
 
-## 5 Read More
+## 5 Limitations
+
+Running multiple synchronization processes at the same time is not supported, regardless the type (full/selective).
+
+If you try to trigger another synchronization process while the synchronization is in progress, the error message will be shown:
```
Comment (omnajjar) [inline content/refguide/synchronize.md:75]: 🎄 I think the word `following` is missing here, so it should be: 

>... progress, the _following_ error message will be shown:

**Review by omnajjar — CHANGES_REQUESTED** [review]

**Review by omnajjar — CHANGES_REQUESTED** [review]

File: content/refguide/offline-first.md (hunk 1 above)
Comment (govorov) [reply content/refguide/offline-first.md]: Good point. Fixed :coffee:

File: content/refguide/synchronize.md (hunk 2 above)
Comment (govorov) [reply content/refguide/synchronize.md]: Fixed :coffee:

File: content/refguide8/offline-first.md [hunk 3]
```diff
@@ -116,11 +116,21 @@ If you have custom widgets or JavaScript actions which use an entity that cannot
 
 {{% image_container width="450" %}}![custom synchronization](attachments/offline-first/custom-sync.png){{% /image_container %}}
 
-### 2.5 Error Handling {#error-handling}
+### 2.5 Limitations
+
+Running multiple synchronization processes at the same time is not supported, regardless the type (full/selective).
```
[the remark this answers is not in the document]
Comment (govorov) [reply content/refguide8/offline-first.md]: :heavy_check_mark:

File: content/refguide8/synchronize.md [hunk 4]
```diff
@@ -68,7 +68,17 @@ The **Action** section of the properties pane shows the action associated with t
 
 {{% snippet file="refguide8/microflow-common-section-link.md" %}}
 
-## 5 Read More
+## 5 Limitations
+
+Running multiple synchronization processes at the same time is not supported, regardless the type (full/selective).
+
+If you try to trigger another synchronization process while the synchronization is in progress, the error message will be shown:
```
[the remark this answers is not in the document]
Comment (govorov) [reply content/refguide8/synchronize.md]: :heavy_check_mark:

**Review by omnajjar — APPROVED** [review]

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**